modules/vaos/app/schemas/appointments.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "required": [
+    "data"
+  ],
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "id",
+          "identifier",
+          "kind",
+          "status",
+          "location_id",
+          "clinic",
+          "start",
+          "cancellable",
+          "extension"
+        ],
+        "additionalProperties": false,
```

Comment:
I believe one of the reasons schema validations have not been helpful in our specs is that we haven't been using `additionalProperties: false`. Note that it has to be added to nested objects as well.